roscpp_azure_iothub/README.md

```diff
@@ -91,7 +98,28 @@ And add the `ros_relays` block to the device twin which you are about to connect
 ``` bash
 az iot hub monitor-events --hub-name <YourIoTHubName> --output table
 ```
-
+You should also see the `reported` properties section updated in your device twin each time `/initialpose` is published to:
+(Note: The message recieved from `/initialpose` below is consolidated for brevity)
+``` json
+{
+    "deviceId": "devA",
+    "properties": {
+        "desired": {
+        },
+        "reported": {
+            "ros_messages": {
+                "/initialpose": [
+                    "/initialpose/header/seq = 39.000000",
```

Comment:
just curious can it be something like `"/initialpose/header/seq": 39.00000` or `"/initialpose/header/seq": "39.00000"` which is more JSON style serialization?